### ISSUE TITLE
Disable LangSmith

### ISSUE THREAD

**Issue by austinmw** [explodinggradients/ragas#238]
Hi, I'm getting the following errors logged when running eval:

```
Connection error caused failure to post http://localhost:1984/runs  in LangSmith API. Please confirm your LANGCHAIN_ENDPOINT. ConnectionError(MaxRetryError("HTTPConnectionPool(host='localhost', port=1984): Max retries exceeded with url: /runs (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x2d0c00d90>: Failed to establish a new connection: [Errno 61] Connection refused'))"))
Connection error caused failure to patch http://localhost:1984/runs/8703dda5-af15-4f7c-adeb-9636a07f847b  in LangSmith API. Please confirm your LANGCHAIN_ENDPOINT. ConnectionError(MaxRetryError("HTTPConnectionPool(host='localhost', port=1984): Max retries exceeded with url: /runs/8703dda5-af15-4f7c-adeb-9636a07f847b (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x2d0bdd640>: Failed to establish a new connection: [Errno 61] Connection refused'))"))
```

How can I disable LangSmith to prevent these?

**Comment by jjmachan** [comment]
unfortunatly there is no fix as of now and it is an annoying set of error messages. We will change that shortly

for now, you can disable the logs if you want 
```py
import logging
logging.getLogger('langchain.callbacks.tracers.langchain').setLevel(logging.WARNING)
logging.getLogger('urllib3.connectionpool').setLevel(logging.ERROR)
```

will put a long term solution in the mean time

**Comment by austinmw** [comment]
Thanks a lot! 🙏🏻

**Comment by ranjanritesh89** [comment]
I also faced the same issue and using the given suggestion to disable the log for now. Thanks @jjmachan

**Comment by jjmachan** [comment]
applogies for this annoyance @ranjanritesh89 but we will also have a permanent fix with #394 too
